Update venice.lua

BlasSoriano opened the pull request:
Added map descriptions and disabled landfill to prevent cheating

BlasSoriano committed:
Update venice.lua

Added map descriptions and disabled landfill to prevent cheating

grilledham commented:
Looks good to me.

The luacheck errors will need fixing though. Looks like it's just trailling whitespace.

BlasSoriano committed:
Update venice.lua

Removed trailing whitespaces (inside description strings, so picky the luacheck).

grilledham commented:
Thanks for the PR 👍
